Src/FluentAssertions/Formatting/DateTimeOffsetValueFormatter.cs

```diff
@@ -70,12 +72,19 @@ public void Format(object value, FormattedObjectGraph formattedGraph, Formatting
                 formattedGraph.AddFragment(" +");
                 formatChild("offset", dateTimeOffset.Offset, formattedGraph);
             }
-
-            if (dateTimeOffset.Offset < TimeSpan.Zero)
+            else if (dateTimeOffset.Offset < TimeSpan.Zero)
             {
                 formattedGraph.AddFragment(" ");
```

Comment:
Shouldn't this have a minus sign?
```suggestion
                formattedGraph.AddFragment(" -");
```

Reply:
I was confused by that one too, but indeed, it's added by the formatter.

Yes, there's test coverage.